node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -58,14 +61,20 @@ type WaitForMpathResult struct {
 }
 
 var (
-	TimeOutMultipathCmd  = 60 * 1000
-	TimeOutMultipathdCmd = 10 * 1000
-	TimeOutBlockDevCmd   = 10 * 1000
-	TimeOutSgInqCmd      = 3 * 1000
+	TimeOutMultipathCmd                  = 60 * 1000
+	TimeOutMultipathdCmd                 = 10 * 1000
+	TimeOutBlockDevCmd                   = 10 * 1000
+	TimeOutSgInqCmd                      = 3 * 1000
+	MultipathdFilterMpathAndVolumeId     = []string{"%w", "%d"}
+	multipathdFilterMpathNameAndVolumeId = []string{"%n", "%w"}
 )
 
 const (
 	DevPath                     = "/dev"
+	DevMapperPath               = "/dev/mapper"
+	ConnectionTypeNVMEoFC       = "nvmeofc"
+	ConnectionTypeFC            = "fc"
+	ConnectionTypeISCSI         = "iscsi"
```

Comment:
please use the [common config](https://github.com/IBM/ibm-block-csi-driver/blob/e60dacf87d9aff2e9544d44dea499f53e9811608/common/config.yaml#L28-L30) for this

Reply:
removed them